cmd/create/execution_util.go

```diff
@@ -0,0 +1,263 @@
+		t, err = cmdCtx.AdminClient().GetTask(ctx, &admin.ObjectGetRequest{
+			Id: &core.Identifier{
+				ResourceType: core.ResourceType_LAUNCH_PLAN,
+				Project:      executionConfig.SourceProject,
+				Domain:       executionConfig.SourceDomain,
+				Name:         lpName,
+				Version:      executionConfig.Version,
+			},
+		})
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		var tList *admin.TaskList
+		tList, err = cmdCtx.AdminClient().ListTasks(ctx, &admin.ResourceListRequest{
+			Limit: 1,
+			Id: &admin.NamedEntityIdentifier{
+				Project: executionConfig.SourceProject,
+				Domain:  executionConfig.SourceDomain,
+				Name:    lpName,
+			},
+			SortBy: &admin.Sort{
+				Key:       "created_at",
+				Direction: admin.Sort_DESCENDING,
+			},
+		})
+		if err != nil {
+			return nil, err
+		}
+		if len(tList.Tasks) == 0 {
+			return nil, fmt.Errorf("no tasks retrieved for %v", lpName)
+		}
+		t = tList.Tasks[0]
+	}
+	return t, nil
+}
+
+func getParamMapForWorkflow(lp *admin.LaunchPlan) (map[string]interface{}, error) {
+	paramMap := make(map[string]interface{})
+	for k, v := range lp.Spec.DefaultInputs.Parameters {
+		varTypeValue, err := coreutils.MakeDefaultLiteralForType(v.Var.Type)
+		if err != nil {
+			fmt.Println("error creating default value for literal type ", v.Var.Type)
+			return nil, err
+		}
+		if paramMap[k], err = coreutils.ExtractFromLiteral(varTypeValue); err != nil {
+			return nil, err
+		}
+		// Override if there is a default value
+		if paramsDefault, ok := v.Behavior.(*core.Parameter_Default); ok {
+			if paramMap[k], err = coreutils.ExtractFromLiteral(paramsDefault.Default); err != nil {
+				return nil, err
+			}
+		}
+	}
+	return paramMap, nil
+}
+
+func getParamMapForTask(task *admin.Task) (map[string]interface{}, error) {
+	paramMap := make(map[string]interface{})
+	for k, v := range task.Closure.CompiledTask.Template.Interface.Inputs.Variables {
+		varTypeValue, err := coreutils.MakeDefaultLiteralForType(v.Type)
+		if err != nil {
+			fmt.Println("error creating default value for literal type ", v.Type)
+			return nil, err
+		}
+		if paramMap[k], err = coreutils.ExtractFromLiteral(varTypeValue); err != nil {
+			return nil, err
+		}
+	}
+	return paramMap, nil
+}
+
+func writeExecConfigToFile(fileName string) error {
+	d, err := yaml.Marshal(executionConfig)
+	if err != nil {
+		fmt.Printf("error: %v", err)
+	}
+	err = ioutil.WriteFile(fileName, d, 0600)
+	if err != nil {
+		return fmt.Errorf("unable to write in %v yaml file", executionConfig.File)
+	}
+	return nil
+}
+
+func readExecConfigFromFile(fileName string) (*ExecutionConfig, error) {
+	data, _err := ioutil.ReadFile(fileName)
+	if _err != nil {
+		return nil, fmt.Errorf("unable to read from %v yaml file", executionConfig.File)
+	}
+	executionConfigRead := ExecutionConfig{}
+	if _err = yaml.Unmarshal(data, &executionConfigRead); _err != nil {
+		return nil, _err
+	}
+	return &executionConfigRead, nil
+}
+
+func askForConfirmation(s string) bool {
```

Comment:
move to common utlity in flytectl, this is a nice function

Reply:
Done. Added commandutils